apisix/init.lua

```diff
@@ -806,14 +806,14 @@ end
 
 function _M.stream_ssl_phase()
     local ngx_ctx = ngx.ctx
-    local api_ctx = ngx_ctx.api_ctx
-
-    if api_ctx == nil then
-        api_ctx = core.tablepool.fetch("api_ctx", 0, 32)
-        ngx_ctx.api_ctx = api_ctx
-    end
+    local api_ctx = core.tablepool.fetch("api_ctx", 0, 32)
+    ngx_ctx.api_ctx = api_ctx
```

Comment:
Seems we can remove this if `ngx_ctx.api_ctx` will eventually be assigned as `nil` ?

Reply:
It is possible to fetch the api_ctx via ngx.ctx if it is not passed as an argument.